drawSvg/raster.py

```diff
@@ -1,11 +1,18 @@
 
 import io
+import warnings
+from .missing import MissingModule
+
 
 try:
     import cairosvg
-except (ImportError, OSError):
-    import warnings
-    from .missing import MissingModule
+except OSError as e:
+    msg = e.args[0]
+    if msg.startswith("no library called \"cairo\" was found"):
+        msg = 'Cairo will need to be installed to rasterize images: See prerequisites section in `README` at https://github.com/cduck/drawSvg#prerequisites'
```

Comment:
I wouldn't even mention "README".
```suggestion
        msg = 'Cairo will need to be installed to rasterize images: See https://github.com/cduck/drawSvg#prerequisites'
```

Reply:
Will change.